### DISCUSSION TITLE
Please tell me your favorite bands

### DISCUSSION THREAD

**Discussion by tk118** [NAIST-SE/AttractingContributors#151] [General]
Please tell me your favorite bands
https://github.com/tk118/AttractingContributors-Band

**Comment by nagata-atsuki** [comment]
added 
Please help me
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Comment by yangziwei96** [comment]
added! and plz help me too
https://github.com/yangziwei96/AttractingContributors-sports
